### ISSUE TITLE
Text argument not recognized

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#6289]
### Skript/Server Version

```markdown
> sk version
[17:39:38 INFO]: Invalid argument 'version'. Allowed are: /skript ...
[17:39:38 INFO]:   reload - Reloads a specific script, all scripts, the config, or everything
[17:39:38 INFO]:   enable - Enables all scripts or a specific one
[17:39:38 INFO]:   disable - Disables all scripts or a specific one
[17:39:38 INFO]:   update - Check for updates, read the changelog, or download the latest version of Skript
[17:39:38 INFO]:   info - Prints a message with links to Skript's aliases and documentation
[17:39:38 INFO]:   help - Prints this help message. Use '/skript reload/enable/disable/update' to get more info
```

### Bug Description

Text argument is not recognized. It was recognized in prior versions.

### Expected Behavior

Clean start-up

### Steps to Reproduce

- Upgrade to Skript 2.8.0-pre1
- Start server
- Reload scripts
- Read console/log

### Errors or Screenshots

```
[17:31:00 INFO]: Line 7: (shutdown.sk)
[17:31:00 INFO]:     Unknown type 'text="We'll be right back!"'
[17:31:00 INFO]:     Line: command /shutdown [<number=120>] [<text="We'll be right back!">]:
```

### Other

shutdown.sk
```
command /shutdown [<number=120>] [<text="We'll be right back!">]:
...
```

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by APickledWalrus** [comment]
Should be fixed in https://github.com/SkriptLang/Skript/pull/6286

**Comment by SlimeDog** [comment]
This issue appears to be resolved in Skript 2.8.0-pre2. Thanks.
